src/components/postgresConnect.ts

```diff
@@ -551,6 +551,18 @@ class PostgresDB {
     }
   }
 
+  public setReviewAssignmentIsLocked = async (reviewAssignmnetId: number, isLocked: boolean) => {
+    const text =
+      'INSERT into review_assignment is_locked VALUES ($2) WHERE id = $1 RETURNING id, is_locked'
+    try {
+      const result = await this.query({ text, values: [reviewAssignmnetId, isLocked] })
+      return result.rows[0]
+    } catch (err) {
+      console.log(err.message)
+      throw err
+    }
+  }
+
```

Comment:
Should probably keep this in the databaseMethods file of the plugin for consistency (unless this is used/intended to be used elsewhere?)

Reply:
Never mind, I see you already did this in your next PR :)